src/quicklist.c

```diff
@@ -3124,6 +3148,52 @@ int quicklistTest(int argc, char *argv[], int accurate) {
         quicklistRelease(ql);
     }
 
+    if (accurate) {
```

Comment:
the first test will take 1 gb of RAM (32*32), and the second one 4gb right?
i think that instead of the `--accurate` we need to add a `--large-memory` and have it disabled by default.
our current actions runners can't handle such big tests, and we documented that tests requiring more than 100mb will be skipped by default (we'll add a self-hosted runner soon to run these)